### PR TITLE
Consistent naming of visualization

### PR DESCRIPTION
**Pull request by hubertp**
Making sure we are consistently using the American spelling of visualization.

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
I am surprised there is just so little of files with wrong names. I see far more of them. For example:
```bash
enso$ find . | grep Visualisation.*scala
./engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/VisualisationOperationsTest.scala
./engine/language-server/src/test/scala/org/enso/languageserver/websocket/binary/VisualisationProtocolTest.scala
./engine/language-server/src/main/scala/org/enso/languageserver/protocol/binary/factory/VisualisationUpdateFactory.scala
./engine/language-server/src/main/scala/org/enso/languageserver/runtime/VisualisationApi.scala
./engine/language-server/src/main/scala/org/enso/languageserver/runtime/handler/AttachVisualisationHandler.scala
./engine/language-server/src/main/scala/org/enso/languageserver/runtime/handler/DetachVisualisationHandler.scala
./engine/language-server/src/main/scala/org/enso/languageserver/runtime/handler/ModifyVisualisationHandler.scala
./engine/language-server/src/main/scala/org/enso/languageserver/runtime/VisualisationConfiguration.scala
./engine/language-server/src/main/scala/org/enso/languageserver/requesthandler/visualisation/AttachVisualisationHandler.scala
./engine/language-server/src/main/scala/org/enso/languageserver/requesthandler/visualisation/DetachVisualisationHandler.scala
./engine/language-server/src/main/scala/org/enso/languageserver/requesthandler/visualisation/ModifyVisualisationHandler.scala
```

**Comment by hubertp** [comment]
> I am surprised there is just so little of files with wrong names. I see far more of them. For example:

I wasn't finished. Hence the draft state.

**Review by GregoryTravis — APPROVED** [review]

**Comment by hubertp** [comment]
In the interest of making (my) life easier could I get a quick yolo :+1: from everyone required? I'd like to avoid dealing with merge conflicts on this one.

**Review by wdanilo — APPROVED** [review]

**Review by jdunkerley — APPROVED** [review]

**Review by Akirathan — APPROVED** [review]

File: app/gui/controller/engine-protocol/src/binary/serialization.rs [hunk 1]
```diff
@@ -531,12 +531,12 @@ impl<'a> DeserializableUnionField<'a, OutboundMessage<'a>> for FromServerPayload
                 })
             }
             OutboundPayload::SUCCESS => Ok(FromServerPayloadOwned::Success {}),
-            OutboundPayload::VISUALISATION_UPDATE => {
-                let payload = message.payload_as_visualisation_update().unwrap();
-                let context = payload.visualisationContext();
+            OutboundPayload::VISUALIzATION_UPDATE => {
+                let payload = message.payload_as_visualization_update().unwrap();
+                let context = payload.visualizationContext();
                 Ok(FromServerPayloadOwned::VisualizationUpdate {
```
Comment (radeusgd) [inline app/gui/controller/engine-protocol/src/binary/serialization.rs]: This does not look good. The casing is all messed up.

The regex used to do this replace should properly inherit the case of the source letter.

**Review by 4e6 — APPROVED** [review]
Yolo :shipit:

**Review by jdunkerley — APPROVED** [review]

**Comment by wdanilo** [comment]
> Yolo :shipit:

😂

**Pull request merged**